WhiteDentalClinic/WhiteDentalClinic.DataAccess/Entities/AppointmentEntity/Appointment.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Runtime.Serialization;
+using WhiteDentalClinic.DataAccess.Entities.CustomerEntity;
+using WhiteDentalClinic.DataAccess.Entities.DentistEntity;
+
+namespace WhiteDentalClinic.DataAccess.Entities.AppointmentEntity
+{
+    public class Appointment
+    {
+        public Guid Id { get; set; }
+        public DateTime? dateTime { get; set; }
```

Comment:
`DateTime` if it's not working find another name but it must be capital first letter as all the other properties.